Dalamud/Game/ClientState/ClientState.cs

```diff
@@ -428,6 +486,8 @@ internal ClientStatePluginScoped()
     void IInternalDisposableService.DisposeService()
     {
         this.clientStateService.TerritoryChanged -= this.TerritoryChangedForward;
+        this.clientStateService.MapChanged -= this.MapChangedForward;
+        this.clientStateService.PublicInstanceChanged -= this.PublicInstanceChangedForward;
```

Comment:
These need cooresponding `this.clientStateService.MapChanged = null`

statements below to ensure that the objects get GC'd, we had an issue with this in the past which is why we always try to set the delegates to null afterwards.